mock-core-configs/etc/mock/templates/openeuler-22.03.tpl

```diff
@@ -27,44 +27,47 @@ protected_packages=
 module_platform_id=platform:oe2203
 user_agent={{ user_agent }}
 
+# all openEuler LTS release will continue developing and releasing SPx version
+# such as 22.03-LTS -> 22.03-LTS-SP1 -> 22.03-LTS-SP2 ...
+# all LTS-SPx is compatible with its LTS release
```

Comment:
Humble feedback :-) can't you have a server-side symbolic links?  Say `openEuler-22.03-latest` pointing at `openEuler-22.03-LTS-SP1`, later when available pointing at `22.03-LTS-SP2`, etc..?  That way your users can keep using the same `*-latest` baseurl= option (including Mock).

Reply:
Big agree with you, its also a maintaince burden for us, will make a suggestion to our release team :)

Reply:
> Humble feedback :-) can't you have a server-side symbolic links? Say `openEuler-22.03-latest` pointing at `openEuler-22.03-LTS-SP1`, later when available pointing at `22.03-LTS-SP2`, etc..? That way your users can keep using the same `*-latest` baseurl= option (including Mock).

Big agree with you, it's also a burden for us, will make a discussion with release team 👍

Reply:
+1

Reply:
Anyway, thank you for explaining the release cycle here!

Does the +1 mean this is reviewed?

Reply:
Yes, please make ti merged.